source/mir/math/stat.d

```diff
@@ -58,6 +86,37 @@ template mean(Summation summation = Summation.appropriate)
             return s / cast(sumType!Range) length;
         }
     }
+    
+    ///
+    @safe @fmamath F
+    mean(Range, F)(Range r, F seed)
```

Comment:
What is a use case for `seed`?

Reply:
As in sum, it can be used to provide additional accuracy. 

However, it would be rare to give it a value other than 0 as it is not an online mean estimate. For instance, if you have a mean and want to update it with new data, this would require also providing the number of elements associated with the initial mean. Thinking on it now, this actually is do-able. We could make the initial number of elements is set up as a default parameter with value of zero. This would enable something like this 
```
auto x = [1.0, 2];
auto y = mean(x); // y = 1.5
auto z = mean([3, 4, 5], y, x.length); // z = 3.0
```

Also you can do
```
auto x = mean([1, 2]); \\ x = 1
float seed = 0;
auto y = mean([1, 2], seed); \\ y = 1.5
```
without needing to cast the original range. 

seed is used in phobos
https://dlang.org/library/std/algorithm/iteration/mean.html

Up to you.

Reply:
Then the `Summator` like API is preferred over seeds. 
See also https://github.com/libmir/mir-algorithm/pull/238#issuecomment-618158406

Reply:
we need either `MeanAccumulator` or a simple fix like #239

Reply:
The benefit of `Summator` / `MeanAccumulator` is that they can work with custom databases without loose of precision since they store internal sum representation without rounding.